client/dashboard/sites/settings-agency-fully-managed/index.tsx

```diff
@@ -0,0 +1,170 @@
+import { DataForm } from '@automattic/dataviews';
+import { useMutation, useQuery } from '@tanstack/react-query';
+import { notFound } from '@tanstack/react-router';
+import {
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	Button,
+	Card,
+	CardBody,
+	CheckboxControl,
+	ExternalLink,
+} from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { store as noticesStore } from '@wordpress/notices';
+import { useState } from 'react';
+import {
+	agencyBlogQuery,
+	siteQuery,
+	siteSettingsMutation,
+	siteSettingsQuery,
+} from '../../app/queries';
+import Notice from '../../components/notice';
+import PageLayout from '../../components/page-layout';
+import SettingsPageHeader from '../settings-page-header';
+import type { Site, SiteSettings } from '../../data/types';
+import type { Field, SimpleFormField } from '@automattic/dataviews';
+
+export function canUpdateA4AFullyManagedSetting( site: Site ) {
+	return site.is_wpcom_atomic;
+}
+
+const fields: Field< SiteSettings >[] = [
+	{
+		id: 'is_fully_managed_agency_site',
+		label: __( 'Allow clients to use the WordPress.com Help Center and hosting features.' ),
+		Edit: ( { field, onChange, data, hideLabelFromVision } ) => {
```

Comment:
I believe we can now just use `Edit: 'checkbox'` here.

Reply:
Unfortunately `Edit: 'checkbox'` didn't meet the use case here where the check status is inverse to the setting value. That is, when is_fully_managed_agency_site is true, the checkbox should not be ticked. Is there a way to do that with the `Edit: 'checkbox'` abstraction?

Reply:
Ah, I see! It's very easy to miss! 🤦 

Hmm, maybe we can introduce a derived field which is the inverse of the current field... something like this:

```js
const [ formData, setFormData ] = useState( {
	is_not_fully_managed_agency_site: ! siteSettings?.is_fully_managed_agency_site,
} );
```

I think that's fine because the current behavior is "worse". For example, the id naming and the label are actually contradictory below, which might cause future confusion:

```js
id: 'is_fully_managed_agency_site',
label: __( 'Allow clients to use the WordPress.com Help Center and hosting features.' ),
```